lib/features/login/presentation/login_view.dart

```diff
@@ -201,25 +201,15 @@ class LoginView extends BaseLoginView {
   }
 
   Widget _buildLoadingProgress(BuildContext context) {
-    return Obx(() => controller.viewState.value.fold(
-      (failure) {
-        switch (controller.loginFormType.value) {
-          case LoginFormType.dnsLookupForm:
-          case LoginFormType.baseUrlForm:
-            return _buildNextButtonInContext(context);
-          case LoginFormType.passwordForm:
-          case LoginFormType.credentialForm:
-            return _buildLoginButtonInContext(context);
-          default:
-            return const SizedBox.shrink();
-        }
-      },
-      (success) {
-        if (success is LoadingState) {
-          return _supportScrollForm(context)
-            ? const HorizontalProgressLoadingButton()
-            : _buildExpandedButton(context, const HorizontalProgressLoadingButton());
-        } else {
+    return Obx(() {
+      if (controller.oidcRetryEnabled.value) {
+        return buildRetryButton(context, onRetry: () {
+          controller.retryCheckOidc();
+        });
+      }
```

Comment:
how about to add a separate `form type: oidcRetry`?

Reply:
Done

Reply:
should be separate to a specific `LoginFormType`

Reply:
Done